scripts/update.ts

```diff
@@ -1,90 +1,114 @@
-import * as semver from "https://deno.land/x/semver@v1.4.1/mod.ts";
-import {
-  lookup,
-  REGISTRIES,
-} from "https://denopkg.com/hayd/deno-udd@0.8.2/registry.ts";
-import { parse } from "https://deno.land/std@0.204.0/flags/mod.ts";
-import { join } from "https://deno.land/std@0.204.0/path/mod.ts";
-import { stringifyForWrite } from "../utils/json.ts";
+import * as semver from 'https://deno.land/x/semver@v1.4.1/mod.ts'
+import { lookup, REGISTRIES } from 'https://denopkg.com/hayd/deno-udd@0.8.2/registry.ts'
+import { parse } from 'https://deno.land/std@0.204.0/flags/mod.ts'
+import { join } from 'https://deno.land/std@0.204.0/path/mod.ts'
+import { stringifyForWrite } from '../utils/json.ts'
+import { parse as jsoncParse } from 'https://deno.land/std@0.204.0/jsonc/mod.ts'
+
+async function exists(path: string) {
+	try {
+		await Deno.stat(path)
+		return true
+	} catch {
+		return false
+	}
+}
 
 // map of `packageAlias` to `packageRepo`
-const PACKAGES_TO_CHECK =
-  /(apps)|(deco)|(\$live)|(deco-sites\/.*\/$)|(partytown)/;
+const PACKAGES_TO_CHECK = /(apps)|(deco)|(\$live)|(deco-sites\/.*\/$)|(partytown)/
 
-interface ImportMap {
-  imports: Record<string, string>;
+type ImportMap = Record<string, string>
+type ImportMapFile = { imports: ImportMap }
+type DenoJson = {
+	imports?: ImportMap
+	importMap?: string
 }
 
-const getImportMap = async (dir: string): Promise<[ImportMap, string]> => {
-  const denoJSONPath = join(dir, "deno.json");
-  const denoJSON = await Deno.readTextFile(denoJSONPath).then(JSON.parse);
-  // inlined import_map inside deno.json
-  if (denoJSON.imports) {
-    return [denoJSON, denoJSONPath];
-  }
-
-  const importMapFile = denoJSON?.importMap ?? "./import_map.json";
-  const importMapPath = join(dir, importMapFile.replace("./", ""));
-  return [
-    await Deno.readTextFile(importMapPath).then(JSON.parse),
-    importMapPath,
-  ];
-};
+const getImportMap = async (dir: string): Promise<[ImportMapFile, string]> => {
+	let denoJsonFile = ''
+
+	if (await exists('deno.json')) {
+		denoJsonFile = 'deno.json'
+	} else if (await exists('deno.jsonc')) {
```

Comment:
you're using single quotes, can you change to use double quotes instead?

Also: I think you should `join(dir, "deno.json")` to use `exists` right?

Reply:
Yep, I forgot to join. I made the commit with the fix